automation/taskcluster/decision_task_nightly.py

```diff
@@ -54,17 +53,14 @@ def generate_build_task(apks, is_staging):
             checkout +
             ' && python automation/taskcluster/helper/get-secret.py'
             ' -s {} -k dsn -f .sentry_token'.format(sentry_secret) +
-            ' && python automation/taskcluster/helper/get-secret.py'
-            '-s {} -k Greenfield -f .adjust_token'.format(adjust_secret) +
             ' && ./gradlew --no-daemon -PcrashReports=true clean test assembleGreenfieldRelease'),
         features={
             "chainOfTrust": True,
             "taskclusterProxy": True
         },
         artifacts=artifacts,
         scopes=[
-            "secrets:get:{}".format(sentry_secret),
-            "secrets:get:{}".format(adjust_secret)
```

Comment:
Out of curiosity, what's the driver for removing the `adjust` secret?

Reply:
I talked directly with Colin for the [original PR](https://github.com/mozilla-mobile/fenix/pull/737), and he mentioned that for builds that aren't released to customers, `adjust_token` shouldn't be provided. For example, local development and "staging" builds won't want `adjust_token`, but nightlies/betas/productions will :)